core/domain/classifier_services.py

```diff
@@ -125,7 +128,12 @@ def handle_trainable_states(
 
     Raises:
         Exception. No classifier algorithm found for the given interaction id.
+
+    Returns:
+        list(ClassifierTrainingJobModel|StateTrainingJobsMappingModel). List of
```

Comment:
Don't just repeat the type in the docstring, explain the conceptual meaning of what is being passed out of the function (i.e. what do these returned things represent).